compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -12681,8 +12681,8 @@ TR::InstOpCode::S390BranchCondition getButestBranchCondition(TR::ILOpCodes opCod
          {
                                  /* Condition Code Value */
     /* branch opcode*/  /*        0            1            2            3       */
-    /* TR::ifiucmpeq */     { TR::InstOpCode::COND_MASK8 , TR::InstOpCode::COND_MASK6 , TR::InstOpCode::COND_MASK0 ,  TR::InstOpCode::COND_MASK1  },
```

Comment:
Just making sure this (and next diff) non-deletion was intentional...?

Reply:
This is intentional -  instead of being deleted, the ifucmpeq and ifiucmpne here should be mapped to their signed ILOpcodes.